content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -160,52 +195,65 @@ plots:
-[DVC Extension]'s [Plots Dashboard].
+To render the plots, run the `dvc plots show` command, which generates an HTML
+file you can then open in a browser:
 
 ```cli
 $ dvc plots show
 file:///Users/dvc/example-get-started/dvc_plots/index.html
 ```
 
+The generated HTML file contains the following rendered plots:
+
 ![](/img/plots_prc_get_started_show.svg)
 ![](/img/plots_roc_get_started_show.svg)
 ![](/img/plots_importance_get_started_show.png '=500 :wrap-left')
 ![](/img/plots_cm_get_started_show.svg)
 
-Let's save this iteration so we can compare it later:
+Alternatively:
+
+- If you're using VSCode's [DVC extension] for your project, you can use the
+  [Plots Dashboard] inside your IDE.
+- If you're using [Studio] for collaboration on your ML project, use the "[Show
+  plots][studio display plots]" button in the project experiment screen. From
+  [Studio], all visualization, comparisons and trends can be easily [shared with
+  others][studio share a project] as well!
+
+Let's save this iteration into Git:
 
 ```cli
 $ git add .gitignore dvc.yaml dvc.lock eval
 $ git commit -a -m "Create evaluation stage"
 ```
 
-Later we will see how to
-[compare and visualize different pipeline iterations](#comparing-iterations).
-For now, let's see how to capture another important piece of information which
-will be useful for comparison: parameters.
+Now, let's see how to capture another critical piece of information for pipeline
+execution: parameters. Tracking parameters along with your pipelines' data is
+essential to making them fully reproducible and for future comparisons and
+analysis to be complete.
 
-[dvclive]: /doc/dvclive
-[`roc_curve`]:
-  https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html
-[`confusion_matrix`]:
-  https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html
 [plots files]: /doc/user-guide/project-structure/dvcyaml-files#plots
-[plots dashboard]:
-  https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/plots.md
 [dvc extension]:
   https://marketplace.visualstudio.com/items?itemName=Iterative.dvc
+[plots dashboard]:
+  https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/plots.md
+[Studio]: /doc/studio
+[studio display plots]:
+  /doc/studio/user-guide/projects-and-experiments/visualize-and-compare#display-plots-and-images
+[studio share a project]:
+  /doc/studio/user-guide/projects-and-experiments/share-a-project#share-a-project
 
 ## Defining stage parameters
 
-It's pretty common for data science pipelines to include configuration files
-that define adjustable parameters to train a model, do pre-processing, etc. DVC
-provides a mechanism for stages to depend on the values of specific sections of
-such a config file (YAML, JSON, TOML, and Python formats are supported).
+It's common for data science pipelines to include configuration files that
```

Comment:
the previous description was better I think ... 

`control code execution and inject external inputs` - this is more about SE, DE. Here we talk about hyperparameters

Reply:
That's the intent. This is about parameters in general and not specifically hyperparameters or even just ML pipelines. These can be any external parameters passed to the code executed, and it was not clear before. it felt restricted to the examples or some specific use cases. Hence the generalization

Reply:
> This is about parameters in general and not specifically hyperparameters

I understand what you were trying to achieve. Yes, we intentionally were avoid that (generalization). We were trying to be careful about our target audience and even though DVC is general enough we were making it specifically for ML audience. Especially in Get Started. The more general you go, more you ask people to bring the general concept to their domain- we were trying to avoid that as much as possible. 

E.g. I know a lot of users / companies that use DVC to store some very unrelated to ML binaries, we have game designers using it, robotic firms to store binary large maps, etc, etc. And yes, we had to make a tradeoff and a bet on what it the primary audience and optimize for it.

Reply:
> Yes, we intentionally were avoid that

Data engineering pipelines are very ML related, but closer to software engineering than data scientists.
DE is the primary context here actually - we're not talking about experiments at all, please note the section 😄

Reply:
yes, we are intentionally not optimizing for DE as well. DVC is not a DE tool (not as a primary, Airflow or Dagster-like at least).

Intention behind including pipelines and metrics (ML, not DE) into data management trail - is to keep each trail self contained and explain all major features (for ML, not DE) in each trail. That might change. My point here that it's not related to DE.

Reply:
this is not about optimizing for DE - but reducing the "optimization for model training" because it went to the point of not explaining what parameters are - the point it, again, to clarify what parameters ARE in DVC 😄 

Parameters are a basic concept of injecting inputs. This is not an ML-only mechanic. it's not called hyper parameters for a reason. DVC does not differentiate between parameters ( model parameters) and hypterparameters ("normal" parameters). there are only parameters for DVC and they could be anything, basically.

I believe the edit here keeps the ML "optimization" (same examples) but with very slight change in wording, to make the description of what parameters are true to the actual mechanism in DVC

Reply:
>  but reducing the "optimization for model training" because it went to the point of not explaining what parameters are - the point it, again, to clarify what parameters ARE in DVC 😄

yep, my point that this optimization is intentional here. We want to optimize for DS/ML cases specifically. We don't want to confuse ppl with general phrases (you asking them to bridge the gap).

Reply:
> you asking them to bridge the gap
 
 I see this the other way around. if the tool is actually giving them a general mechanism, but the words only speak of some specific ML use case, then it is asking them to bridge the gap. here the words HELP them bridge the gap, expecting less "mental leaps" in the process

Reply:
Mo logic is - DS/ML folks more familiar with their domain terminology, patterns, workflows. They can understand a bit more general thing. My it takes time to apply it back to their domain. 

E.g. (just as an example to highlight what I mean) "DVC is about tracking large files" vs "DVC helps tracking ML models". Both are true. Second one more precise for people who are dealing with models.

Reply:
> My it takes time to apply it back to their domain.

I think that by trying to apply ML terminology to everything we might err on the side of forgetting to explain what the tool actually does / gives you. can't skip the "basics" in get-started (especially not with junior DS/ML engineers). This is what causes confusion for readers who don't already know the tools (and what causes people to write so many "basic guides" or docs about DVC). 

> Second one more precise for people who are dealing with models.

Great example - super arguable 😆! I would say this -  2nd is definitely LESS precise. it's not explaining what DVC does, it **applies** what it does for some domain. that is not precision in describing / explaining. that is targeting (from the marketing/sales world). and targeting better != explaining better. this confusion could explain a lot of disagreements that I've seen around the docs and reluctancy to accept some changes.
I firmly believe that clarity and avoiding confusing (good docs) should be 1st priority.

We should be careful not to mix those (explain X, apply X to ML) - It's confusing for people to read about application of something that wasn't defined / described in any way before.

A very concrete example from this page - In the current version, there's no mention that, for DVC, metrics need to be written into dedicated files ! 😱 
In the real world that is a very non-trivial assumption to make - metrics are often tracked in memory only, and not persisted "raw", but only plotted at the end of the run. Alternatively, more advanced set-ups are writing metrics to a proper database (of some sort).
Both of these won't work with DVC. This is the sort of things that need some explanation. I think that for people who've been working with DVC for a long time, it's easy to start assuming a lot of things as obvious, but we're unintentionally "forcing" new-comers to do "mental leaps" we can easily avoid / minimize